### DISCUSSION TITLE
Sandboxxing

### DISCUSSION THREAD

**Discussion by trymeouteh** [waydroid/waydroid#473] [Q&A]
Can the Android OS be sandboxxed from the rest of your Linux system, similar to how Flatpaks can be sanboxxed with the FlatSeal app to make Flatpaks permissions?

Settings to prevent Waydroid bottles from accessing your webcam, microphone, bluetooth, internet, files, etc?

**Comment by RaptaG** [comment]
I was about to ask the same thing

**Comment by TriMoon** [comment]
Waydroid is already using LXC to run inside a container, so it's already "sandboxed"...

**Reply by RaptaG** [reply]
Really? That's awesome!

**Reply by iconoclasthero** [reply]
If it is so containerized, how come it is so hard to remove from one's system?  I was unable to remove it from Ubuntu 22.04 and it wasn't completely gone until I created a new partition and installed Rhino. 

nb: I found this thread from a search of sandboxing waydroid to try to find some way to wall it off and make it simple to remove.
